### DISCUSSION TITLE
Couldn't find an answer for this on forums.

### DISCUSSION THREAD

**Discussion by Joesk** [Truinto/DarkCodex#73] [General]
Was wondering if this works with or conflicts with Toybox and the Tabletop tweaks mods? I noticed there appears to be some overlap of abilities with tweaks.
  I like a few things this adds and want to try it out.

  Thank you for your work.

**Comment by Truinto** [comment]
I use both of these mods. No issues. Besides I have a high emphasis on compatibility. The modding system is quite good with it. The worst thing that could happen is that some feature does not work and does nothing.
